app/main.py

```diff
@@ -15,5 +15,11 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
-
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+variables = [lucky_number, pi, one_is_a_prime_number, name,
+             my_favourite_films, profile_info, marks, collection_of_coins]
+for i in range(len(variables)):
```

Comment:
you don't need loop or function, just put data manually in dict

Reply:
sorted_variables = {"mutable": [lucky_number, ....], "immutable": [my_favourite_films, ...]}